modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/upgrade/OnboardingUpgradeFeaturesPage.kt

```diff
@@ -672,6 +652,53 @@ private fun OldFeatureItem(
     }
 }
 
+@Composable
+fun BoxWithConstraintsScope.NoSubscriptionsLayout(
+    onBackPressed: () -> Unit,
+    onNotNowPressed: () -> Unit,
+) {
+    Column(
+        Modifier
+            .windowInsetsPadding(WindowInsets.statusBars)
+            .windowInsetsPadding(WindowInsets.navigationBars)
+            .heightIn(min = this.calculateMinimumHeightWithInsets())
+            .fillMaxWidth()
+    ) {
+
+        Spacer(Modifier.height(8.dp))
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            horizontalArrangement = Arrangement.SpaceBetween,
+            verticalAlignment = Alignment.CenterVertically,
+        ) {
+            NavigationIconButton(
+                onNavigationClick = onBackPressed,
+                iconColor = Color.White,
+                modifier = Modifier
+                    .height(48.dp)
+                    .width(48.dp)
+            )
+            TextH30(
+                text = stringResource(LR.string.not_now),
+                color = Color.White,
+                modifier = Modifier
+                    .padding(horizontal = 24.dp)
+                    .clickable { onNotNowPressed() },
+            )
```

Comment:
I think we shouldn't have both of these buttons when the user directly navigates to this page (i.e., by clicking on the folder icon). Both buttons make sense in the onboarding flow though.

Reply:
You're right. I updated it in 8c73fef.